### ISSUE TITLE
Native build from gem silently fails if python is absent

### ISSUE THREAD

**Issue by jscheid** [cowboyd/libv8#11]
Hi, I ran into an issue where `gem install libv8` would succeed without any error messages but didn't build the native libraries.

```
$ gem install libv8
Building native extensions.  This could take a while...
Successfully installed libv8-3.3.10.2
1 gem installed
Installing ri documentation for libv8-3.3.10.2...
unable to convert "\xE4" to UTF-8 in conversion from ASCII-8BIT to UTF-8 to US-ASCII for lib/libv8/scons/CHANGES.txt, skipping
unable to convert "\xE4" to UTF-8 in conversion from ASCII-8BIT to UTF-8 to US-ASCII for lib/libv8/build/scons/CHANGES.txt, skipping
Installing RDoc documentation for libv8-3.3.10.2...
unable to convert "\xE4" to UTF-8 in conversion from ASCII-8BIT to UTF-8 to US-ASCII for lib/libv8/scons/CHANGES.txt, skipping
unable to convert "\xE4" to UTF-8 in conversion from ASCII-8BIT to UTF-8 to US-ASCII for lib/libv8/build/scons/CHANGES.txt, skipping
$ find ~/.rvm/gems/ruby-1.9.2-p180/gems/libv8-3.3.10.2 -name \*.a
$
```

I decided to follow the instructions in your readme and build it from a git clone, which complained about python absent (this is on a bare colo server with a minimal install):

```
$ bundle exec rake compile
(in /tmp/libv8/libv8)
Previous HEAD position was de15804... Version 3.3.10
HEAD is now at f985a1d... Version 3.4.4
Compiling V8...
make: python: Command not found
make: *** [build/scons/install/bin/scons] Error 127
$
```

So I installed python and reinstalled the gem, and this time it installed the libraries:

```
$ find ~/.rvm/gems/ruby-1.9.2-p180/gems/libv8-3.3.10.2 -name \*.a
/home/foo/.rvm/gems/ruby-1.9.2-p180/gems/libv8-3.3.10.2/lib/libv8/build/v8/libv8.a
/home/foo/.rvm/gems/ruby-1.9.2-p180/gems/libv8-3.3.10.2/lib/libv8/build/v8/libv8preparser.a
$
```

In summary, it looks like the gem installation swallows the error message about python being absent and continues regardless.  Needless to say it would be nice if the user was told about the problem and the gem installation would fail.
